manifests/charts/global.yaml

```diff
@@ -161,42 +114,6 @@ global:
     # This controls the 'policy' in the sidecar injector.
     autoInject: enabled
 
-    # Sets the destination Statsd in envoy (the value of the "--statsdUdpAddress" proxy argument
-    # would be <host>:<port>).
-    # Disabled by default.
-    # The istio-statsd-prom-bridge is deprecated and should not be used moving forward.
-    envoyStatsd:
```

Comment:
Where is this config now (for those of us that haven't been closely following the install changes)?

Reply:
This is not even used in the install charts anymore, so its nowhere. I think that happened long ago - I vaguely recall over a year ago noticing this was dead code

Reply:
Wow, OK. I was fairly certain there were users for these options. I'm surprised that hasn't come up.